debian/rules

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/make -f
+
+BUILDDIR = build_dir_debian
+
+%:
+	dh $@ --parallel
+
+override_dh_auto_configure:
+	mkdir $(BUILDDIR)
+	cd $(BUILDDIR) && cmake .. -DCMAKE_BUILD_TYPE=Release -DWITH_QT4=1 -DWITH_CLIENT=1 -DWITH_SERVER=1 -DWITH_ORACLE=1 -DCMAKE_INSTALL_PREFIX:PATH=/usr
```

Comment:
@Daenyth what is your thought process behind making it multi-build option?  On one hand I would agree options are nice, but for most users it just adds confusion and for the ones that take the time to look into the differences there is a high likely hood they are going to configure things how they choose to rather than rely on pre-defined  set of config options.